core/src/consensus/signer.rs

```diff
@@ -77,11 +77,29 @@ impl EngineSigner {
         Ok(result)
     }
 
+    /// Sign a message hash with ECDSA.
+    pub fn sign_ecdsa(&self, hash: H256) -> Result<Signature, AccountProviderError> {
```

Comment:
Why not 'hash: &H256`?

Reply:
To keep consistency with the function `sign` in the same file.
I wonder why it was implemented like that though.